src/utils/index.js

```diff
@@ -0,0 +1,19 @@
+import accepts from 'attr-accept'
+
+export const supportMultiple =
```

Comment:
I really prefer `function` declarations for functions unless they are in the clojure. Can you live this that and make me happy again™? ;)